src/Config/ScaleOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ
+{
+    public class ScaleOptions
```

Comment:
This is for runtime driven scale only, right? I think before we do this, we want to have a consistent story for this configuration across all the extensions. I don't believe this is configurable for any of the other extensions currently. Also we'll need to make it clear to users that this is only for RTS and won't affect Consumption scaling, since the ScaleController doesn't currently allow configuration of these. That could be a source of confusion for people. @paulbatum FYI

Reply:
Okay, I'll go ahead and revert this. And I'll open a separate issue for that.

Reply:
Reverted this.

Reply:
Opened issue here https://github.com/Azure/azure-functions-rabbitmq-extension/issues/118